example/3d.js

```diff
@@ -0,0 +1,359 @@
+codeDiv.append(codeLabel);
+codeDiv.append(code);
+document.body.appendChild(codeDiv);
+code.style.width = '600px';
+code.style.height = '60%';
+code.style.display = 'none';
+codeDiv.style.maxHeight = '60%';
+codeDiv.style.position = 'absolute';
+codeDiv.style.right = 0;
+codeDiv.style.bottom = 0;
+codeDiv.style.zIndex = 10;
+codeDiv.style.backgroundColor = 'white';
+codeDiv.style.padding = '10px';
+codeDiv.style.overflow = 'auto';
+codeDiv.style.fontSize = '9px';
+
+codeLabel.addEventListener('click', function () {
+  code.style.display = code.style.display === 'none' ? 'block' : 'none';
+});
+
+var error = document.createElement('span');
+error.style.color = '#cc3333';
+error.style.fontFamily = 'sans-serif';
+error.style.fontStyle = 'italic';
+error.style.position = 'absolute';
+error.style.left = '10px';
+error.style.bottom = '10px';
+error.style.zIndex = '1';
+
+const showError = str => (error.textContent = str);
+const clearError = () => (error.textContent = '');
+document.body.appendChild(error);
+
+require('regl')({
+  onDone: require('fail-nicely')(run),
+  pixelRatio: Math.min(window.devicePixelRatio, 1.5),
+  attributes: {
+    antialias: true,
+    alpha: false
+  }
+});
+
+function run (regl) {
+  var i, j;
+
+  const camera = require('./regl-camera')(regl, {
+    phi: 0.5,
+    theta: 0.2,
+    damping: 0,
+    distance: 15,
+    noScroll: true
+  });
+
+  const state = {
+    hull: true,
+    uDegree: 2,
+    vDegree: 3,
+    uPoints: 6,
+    vPoints: 6,
+    uBoundary: 'closed',
+    vBoundary: 'closed',
+    uFrequency: 4,
+    vFrequency: 4,
+    minorRadius: 1,
+    majorRadius: 3,
+    weightStrength: 0,
+    twist: 0
+  };
+
+  controlPanel([
+    {label: 'uDegree', type: 'range', min: 1, max: 8, step: 1, initial: state.uDegree},
+    {label: 'vDegree', type: 'range', min: 1, max: 8, step: 1, initial: state.vDegree},
+    {label: 'uPoints', type: 'range', min: 3, max: 31, step: 1, initial: state.uPoints},
+    {label: 'vPoints', type: 'range', min: 3, max: 101, step: 1, initial: state.vPoints},
+    {label: 'uBoundary', type: 'select', options: ['closed', 'open', 'clamped'], initial: state.uBoundary},
+    {label: 'vBoundary', type: 'select', options: ['closed', 'open', 'clamped'], initial: state.vBoundary},
+    {label: 'uFrequency', type: 'range', min: 0, max: 8, step: 1, initial: state.uFrequency},
+    {label: 'vFrequency', type: 'range', min: 0, max: 16, step: 1, initial: state.vFrequency},
+    {label: 'minorRadius', type: 'range', min: 0.1, max: 3, step: 0.1, initial: state.minorRadius},
+    {label: 'majorRadius', type: 'range', min: 1, max: 6, step: 0.1, initial: state.majorRadius},
+    {label: 'weightStrength', type: 'range', min: 0, max: 1, step: 0.01, initial: state.weightStrength},
+    {label: 'twist', type: 'range', min: -2, max: 2, step: 0.01, initial: state.twist},
+    {label: 'hull', type: 'checkbox', initial: state.hull}
+  ]).on('input', function (data) {
+    var needsUpdate = false;
```

Comment:
We seem to have started with ES6 `const` and drifted back to ES5 `var` instead of ES6 `let`

Reply:
Yeah, no particular intention behind it. The examples need major cleanup. They were really just a place to do heavy testing/development.

Reply:
Upon further examination, this is intentional. [es2040](https://www.npmjs.com/package/es2040) supports `const` but not `let`.

Reply:
Re: es2040:

> Is this a joke?
> Not really. The TC39 does not represent my interests, and the features they introduce are not useful for the stuff I'm doing. I'm bloody serious. A few good things have been introduced, so that's what we're backporting to older browsers.

LOL.